index.js

```diff
@@ -797,19 +811,17 @@ app.post('/api/door', Validator.body(['coreid', 'data']).exists(), async (reques
         const signal = message.data
         const control = message.control
         let doorSignal
-        if (signal === IM21_DOOR_STATUS.OPEN) {
+        if (signal === IM21_DOOR_STATUS.OPEN || signal === IM21_DOOR_STATUS.HEARTBEAT_OPEN) {
           doorSignal = SESSIONSTATE_DOOR.OPEN
-        } else if (signal === IM21_DOOR_STATUS.CLOSED) {
+        } else if (signal === IM21_DOOR_STATUS.CLOSED || signal === IM21_DOOR_STATUS.HEARTBEAT_CLOSED) {
           doorSignal = SESSIONSTATE_DOOR.CLOSED
         } else if (signal === IM21_DOOR_STATUS.LOW_BATT) {
           doorSignal = 'LowBatt'
-          helpers.logSentry(`Received a low battery alert for ${locationid}: ${message}`)
-        } else if (signal === IM21_DOOR_STATUS.HEARTBEAT_OPEN || signal === IM21_DOOR_STATUS.HEARTBEAT_CLOSED) {
-          doorSignal = 'HeartBeat'
+          helpers.logSentry(`Received a low battery alert for ${locationid}`)
+          BraveAlerter.sendSingleAlert(locationid, `The battery for the ${location.displayName} door sensor is low, and needs replacing.`)
```

Comment:
There is a `sendSingleAlert` function declared locally in `index.js` that takes locationid and a message as parameters. I think you want to use this instead.

The `sendSingleAlert` method in BraveAlerter is more generic and takes the toPhoneNumber, fromPhoneNumber, and message as parameters.

Reply:
Hmmm, on the other hand, the local `sendSingleAlert` function will send the message to the heartbeatRecipients. Is this what we want?

Maybe we want to call `braveAlerter.sendSingleAlert` (note the small "b" to use the already-defined `braveAlerter` object rather than the class), with the responderPhone as the toPhoneNumber.

Reply:
ah, yeah this was a mistake, I want the local sendSingleAlert to text the heartbeat numbers. I'd prefer this over just the responder phone since Heartbeat numbers includes them but will also notify us of the issue which can contextualize response etc